cli/command/manifest/fetch.go

```diff
@@ -0,0 +1,326 @@
+package manifest
```

Comment:
The `cli/command/` packages should only contain the command structure. Since this seems to be more application logic please move this to a `cli/manifest` package so we can keep the application logic separate from the "command routing and arg parsing" that happens in the `cli/command` package.

The same would apply to `fetch_v1.go`, and `fetch_v2.go`. `push.go` probably needs to be split up, as it's rather large, and I expect part of it would end up in `cli/manifest` as well.